src/lib/validApiKey.ts

```diff
@@ -3,10 +3,13 @@ import { ChatOpenAI } from '@langchain/openai'
 
 export const validateApiKey = async (
   openAIApiKey: string,
+  baseURL: string
 ): Promise<boolean> => {
-  const model = new ChatOpenAI({ openAIApiKey })
+  const model = new ChatOpenAI({ openAIApiKey:openAIApiKey,configuration: {
+    baseURL:  baseURL || "https://api.openai.com/v1",
+  },})
   try {
-    await model.invoke([new HumanMessage('Say Ok')])
+    //await model.invoke([new HumanMessage('Say Ok')])
```

Comment:
Why comment this ? we have to validate the api key right ?

Reply:
Oh. I just comment this for testing. Sorry about that